website/content/docs/enterprise/index.mdx

```diff
@@ -8,21 +8,31 @@ description: >-
 
 # Consul Enterprise
 
-[Consul Enterprise](https://www.hashicorp.com/consul) features address the organizational complexities of collaboration, operations, scale and governance. It includes capabilities that improve failure resilience, read scalability, and managing access in accordance with organizational policies. It also adds support for application and service architectures across complex network topologies. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing) below.
+[Consul Enterprise] features address the organizational complexities of
+collaboration, operations, scale and governance.
+It includes capabilities that improve failure resilience, read scalability,
+and managing access in accordance with organizational policies.
+It also adds support for application and service architectures across complex network topologies.
 
-Features include:
+To use Consul Enterprise features,
+[apply a purchased license](/docs/enterprise/license/overview)
+that grants access to the desired features to the Consul Enterprise binary.
+Available enterprise features per Consul variant and license include:
 
-- [Audit Logging](/docs/enterprise/audit-logging)
-- [Automated Backups](/docs/enterprise/backups)
-- [Automated Upgrades](/docs/enterprise/upgrades)
-- [Enhanced Read Scalability](/docs/enterprise/read-scale)
-- [Redundancy Zones](/docs/enterprise/redundancy)
-- [Advanced Federation for Complex Network Topologies](/docs/enterprise/federation)
-- [Network Segments](/docs/enterprise/network-segments)
-- [Namespaces](/docs/enterprise/namespaces)
-- [NIA with Terraform Enterprise](/docs/nia/enterprise)
-- [Sentinel](/docs/enterprise/sentinel)
-- [OIDC Auth Method](/docs/acl/auth-methods/oidc)
+| Feature                                                                           | [HashiCorp Cloud Platform (HCP) Consul]                   | [Consul Enterprise] | Legacy Consul Enterprise (module-based)           |
+| --------------------------------------------------------------------------------- | ------------------------------------------------------- | ----------------- | ------------------------------------------------- |
+| Consul servers as a managed service                                               | Yes                                                     | No (self-managed) | No (self-managed)                                 |
```

Comment:
@pcmccarron : Do you want to keep this table row (in red below)? Anything you'd change about it?

![image](https://user-images.githubusercontent.com/85913323/145654627-f1bbb0f9-f8a2-4815-9bdf-c0b082a793f2.png)